packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-skeleton {
+  border-radius: 2px;
+  background-color: rgba($gray1, 0.2);
+  width: 100%;
+  height: $pt-grid-size * 0.6;
+
+  // negative animation delay ensures animations are synced
+  animation: ($pt-transition-duration * 20) linear ($pt-transition-duration * -70) infinite glow;
+
+  & + & {
+    margin-top: $pt-grid-size;
+  }
+}
+
+@keyframes glow {
+  0% {
+    background-color: rgba($gray4, 0.2);
+  }
+
+  50% {
+    background-color: rgba($gray1, 0.2);
+  }
+
+  100% {
+    background-color: rgba($gray4, 0.2);
+  }
+}
+
+/*
+Skeleton
+
+Skeletons allow you to show a loading state that mimics the shape of your yet-to-load content.
+
+<div class="pt-callout pt-intent-warning pt-icon-warning-sign">
+  <h5>Note</h5>
```

Comment:
this is core documentation. callouts should be used to emphasize something already mentioned, or an edge case. but we haven't mentioned anything yet.

i'd move this content to the CSS API section and expand it because it's essential usage instructions.